### PR TITLE
feat: PGP key (for datenschutz@p0 on privacy policy page + why PGP)

### PR DESCRIPTION
**Pull request by hugurp**
This PR adds the PGP key for the `datenschutz` p0 email address on the privacy policy page (in both languages).

It also adds a link to a FSF infographic, explaining users the reasons, benefits, and how-to of using PGP.

### CODE REVIEW

File: content/privacypolicy.de.md [hunk 1]
```diff
@@ -45,7 +45,13 @@ Hier bekommen Sie auch gerne weitere Informationen.
 
 Die Geltendmachung dieser Rechte ist für Sie kostenlos.
 
-Falls Sie Fragen zum unternehmensinternen Datenschutz haben oder nähere Informationen bezüglich der Datenverarbeitung bei PORT ZERO haben oder Ihr Auskunftsrecht wahrnehmen wollen, kontaktieren Sie uns bitte unter [datenschutz@port-zero.com](mailto:datenschutz@port-zero.com).
+Falls Sie Fragen zum unternehmensinternen Datenschutz haben oder
+nähere Informationen bezüglich der Datenverarbeitung bei PORT ZERO
+haben oder Ihr Auskunftsrecht wahrnehmen wollen, kontaktieren Sie uns
+bitte unter
+[datenschutz@port-zero.com](mailto:datenschutz@port-zero.com)
+(PGP-key: [C56E F7FD 3D3A 34F5 39AA  1304 E0E2 BAEE 4887
```
Comment (hellerve) [inline content/privacypolicy.de.md]: I would format that as `PGP Key`, without the dash and capitalized.

Comment (hugurp) [reply content/privacypolicy.de.md]: Good for me, then I will also change it on this page for homogeneity https://github.com/port-zero/website/blob/master/content/contact.md

Comment (hellerve) [reply content/privacypolicy.de.md]: Yes please!

**Comment by hellerve** [comment]
Thank you!

**Pull request merged**